lib/tasks/populate.rake

```diff
@@ -0,0 +1,14 @@
+require 'faker' 
+require 'factory_bot_rails'
+
+namespace :db do
+
+    desc 'populate dev database with sample student data'
+    
+    task :populate => :environment do
+        Rake::Task['db:reset'].invoke
+        20.times do |n|
+            FactoryBot.create(:student)
+        end
+    end
+end
```

Comment:
Nice idea! this is something we're missing in the plus repo atm (a nice way to generate dev data).

One suggestion would maybe not overload the `db` namespace, maybe use a `dev` or `dev_data` namespace so it runs something like `rake dev:populate` and then store under the `tasks/dev` folder.

This is just my personal opinion though and might not be considered best practice here? I believe it's a good idea since it allows us to clearly see what custom rake tasks are defined.

Reply:
thanks - that feels much cleaner